src/app/icd/ICDNotifier.cpp

```diff
@@ -89,5 +89,16 @@ void ICDNotifier::BroadcastActiveRequestWithdrawal(ICDListener::KeepActiveFlags
     }
 }
 
+void ICDNotifier::BroadcastICDManagementEvent(ICDListener::ICDManagementEvents event)
```

Comment:
Suggest "NotifyICDManagementEvent" since there is no "broadcasting" happening in the usual sense.

Reply:
Prefer leaving `Broadcasting` for now since all the function in the class have the same terminology. A follow up PR could be done to rename the terminology.